cmake/Builtins.cmake

```diff
@@ -17,15 +19,39 @@ endmacro()
 
 if(BUILD_VC)
   list(APPEND BUILTINS Vc)
-  add_subdirectory(builtins/Vc)
+  add_subdirectory(cmake/Vc)
   set(VC ON CACHE BOOL "" FORCE)
 else()
   clear_pkg_vars(Vc)
 endif()
 
 if(BUILD_UMESIMD)
   list(APPEND BUILTINS UMESIMD)
-  add_subdirectory(builtins/umesimd)
+
+  FetchContent_Declare(umesimd
```

Comment:
If the version bump is required only due to the usage of `FetchContent` (seems to be the case, looking at the changes), then I think that it's not really worth it.